app/controllers/concerns/application_notifications_concern.rb

```diff
@@ -1,14 +1,16 @@
 module ApplicationNotificationsConcern
   extend ActiveSupport::Concern
-
-  # Returns the view path of the notification
+  # Loads recent activity feeds of a given course.
   #
-  # @param [Course::Notification|UserNotification] notification The target notification
-  # @return [String] The view path of the notification
-  def notification_view_path(notification)
-    activity = notification.activity
-    root_path = "notifiers/#{activity.notifier_type}/#{activity.event}"
-    notification_class_name = notification.class.name.underscore.gsub('/', '_').pluralize
-    "/#{root_path}/#{notification_class_name}/#{notification.notification_type}"
+  # @param [Course] course The target course.
+  # @param [Integer] count The count of activity feeds that will be loaded. By default it will
+  #   load all activity feeds.
+  # @return [Array<CourseNotification>] Recent activity feed notifications
+  def load_recent_activity_feeds(course, count = :unset)
```

Comment:
uh? Maybe I got a wrong file name. This concern is just included in course controller.